.gitignore

```diff
@@ -35,6 +35,12 @@ provider.hpp
 /*.csv
 /*.csv.meta
 
+# Ignore generated and cached TPC-DS tables
+*.csv
+*.bin
```

Comment:
Oooh no. That applies to all csvs and bins. What's with test data etc.?

Reply:
Sorry, I just mixed up the merge branch.